inst/@sym/display.m

```diff
@@ -136,12 +136,11 @@ function display(x)
   s = [s1 s2];
   n = ustr_length (s);
   %fputs (1, s);  % only in octave, not matlab
-  fprintf (s)
+  disp (s)
```

Comment:
this will break the display of "snippets" because `disp` always puts a newline.  Instead, do the display after the `if (display_snippet)`...